partcad-cli/src/partcad_cli/click/commands/version.py

```
@@ -0,0 +1,12 @@
+import logging
+import partcad as pc
+import partcad_cli as pcc
+import partcad.logging as pc_logging
+import rich_click as click
+
+
+# TODO: @alexanderilyin: Write test which asserts this command is fast
+@click.command(help="Print PartCAD Python Module & PartCAD CLI versions and exit")
+def cli():
+    pc_logging.info("PartCAD Python Module version: %s" % pc.__version__)
```

Ruff (0.8.0)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)